src/components/include/protocol_handler/protocol_handler.h

```diff
@@ -103,6 +103,9 @@ class ProtocolHandler {
   virtual void SendEndService(int32_t connection_id,
                               uint8_t session_id,
                               uint8_t service_type) = 0;
+
+  virtual void NotifyOnFailedHandshake() = 0;
```

Comment:
@AByzhynar description required

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad